control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -2721,6 +2763,8 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/HostNqn'
+        kind:
+          $ref: '#/components/schemas/ReplicaKind'
```

Comment:
should this be optional, or add a default?

Reply:
Making it optional as the grpc layer has this as optional.

Reply:
Added the `Regular` as default for now.